ospfd/ospf_zebra.c

```diff
@@ -1089,10 +1168,14 @@ ospf_zebra_read_ipv4 (int command, struct zclient *zclient,
   else
     api.tag = 0;
 
-  ospf = ospf_lookup ();
+  ospf = ospf_lookup_by_vrf_id (vrf_id);
   if (ospf == NULL)
     return 0;
 
+  if (IS_DEBUG_OSPF_TRACE)
```

Comment:
This is meant for OSPF daemon internal debug activity. If any objections, I will remove it.